logging/fluentd/fluent.conf

```diff
@@ -0,0 +1,14 @@
+<system>
```

Comment:
Was your goal to copy that into the custom fluentd container? I guess a `CP` is missing then in the Dockerfile.